src/components/Pickers/BasePicker.Props.ts

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { IPickerItemProps } from './PickerItem.Props';
+
+export interface IBasePickerProps<T> extends React.Props<any> {
+  /**
+   * Function that specifies how the selected item will appear.
+   */
+  onRenderItem?: (props: IPickerItemProps<T>) => JSX.Element;
+  /**
+   * Function that specifies how the suggestion will appear
+   */
+  onRenderSuggestion?: (props: T) => JSX.Element;
```

Comment:
It seemed weird to me when i was this that this wasn't Suggestions plural. Intended?

The comment is unclear as well. What is "the suggestion"? Do you mean "suggestions"?